<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="BRPC014239" failures="2" tests="4" name="Regression Suite 1" time="0.052" errors="0" timestamp="2022-11-16T11:02:24 BRT">
  <testcase classname="testClasses.TestNG_ListenerTest1" name="testMethod1" time="0.009"/>
  <testcase classname="testClasses.TestNG_ListenerTest1" name="testMethod2" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testClasses.TestNG_ListenerTest1.testMethod2(TestNG_ListenerTest1.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- testMethod2 -->
  <testcase classname="testClasses.TestNG_ListenerTest2" name="testMethod1" time="0.001"/>
  <testcase classname="testClasses.TestNG_ListenerTest2" name="testMethod2" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testClasses.TestNG_ListenerTest2.testMethod2(TestNG_ListenerTest2.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- testMethod2 -->
</testsuite> <!-- Regression Suite 1 -->
